mobify/platform-scaffold
# WEB-1336: Add Filter

**JIRA**: [WEB-1336](https://mobify.atlassian.net/browse/WEB-1336)

## Changes
- Add filter functionality to the product list page

## Todo
- [x] Add the UI part of filters
- [x] Verify typography and spacing
- [x] Verify all requirements have been met from the Jira ticket
- [x] Write tests... #feelShameAsAMinorAction
- [x] Merge #543 
- [x] Fix merge conflicts
- [x] Finish applying feedback

## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Visit a product list page like the [Potions page](https://www.merlinspotions.com/potions.html)
- Verify the Filter functionality works as intended
- Visit a page with the `?filter_is_open` query string present: https://www.merlinspotions.com/potions.html?filter_is_open
- Verify that the page opens with the filter modal open by default
